tests/bgpd/test_peer_attr.py

```diff
@@ -84,10 +84,10 @@ class TestFlag(frrtest.TestMultiOut):
 TestFlag.okfail("peer\\ipv4-multicast\\default-originate")
 TestFlag.okfail("peer\\ipv6-unicast\\default-originate")
 TestFlag.okfail("peer\\ipv6-multicast\\default-originate")
-TestFlag.okfail("peer\\ipv4-unicast\\default-originate route-map")
-TestFlag.okfail("peer\\ipv4-multicast\\default-originate route-map")
-TestFlag.okfail("peer\\ipv6-unicast\\default-originate route-map")
-TestFlag.okfail("peer\\ipv6-multicast\\default-originate route-map")
+# TestFlag.okfail("peer\\ipv4-unicast\\default-originate route-map")
```

Comment:
This is because previously on issuing "no neighbor default-originate route-map <WORD>",
the whole config was getting removed. And the testcase was checking for this removal only.
But with the fix, we are keeping the default-originate config intact, so these testcases are no longer valid.
Hence, commenting those out.

Is there a more appropriate way to handle these testcases?
If yes, could you please let us know/help us out.
Thank you :)

Reply:
test_peer_attr.c has also the same definitions, we might fix all of them maybe?

Reply:
Hi Donatas,
Based on Igor's comment,
we might have to drop this specific commit.
So, the changes in test_peer_attr will no longer be required.
Thank you :)